content/pt/docs/what-is-opentelemetry.md

```diff
@@ -0,0 +1,133 @@
+flexibilidade de que precisam no mundo da computação moderna de hoje.
+
+Se você quiser saber mais, dê uma olhada na
+[missão, visão e valores](/community/mission/) do OpenTelemetry.
+
+## Os principais componentes do OpenTelemetry
+
+OpenTelemetry consiste dos seguintes componentes:
+
+- Uma [especificação](/docs/specs/otel) para todos os componentes
+- Um [protocolo](/docs/specs/otlp/) padrão que defini a forma dos dados de
+  telemetria
+- [Convenções semânticas](/docs/specs/semconv/) que estabelecem um padrão de
+  nomenclatura para tipos comuns de dados de telemetria.
+- APIs que define como gerar dados de telemetria
+- [SDKs de linguagem](/docs/languages) que implementam a especificação, APIs e
+  exportação de dados de telemetria
+- Um [ecossistema de bibliotecas](/ecosystem/registry) que implementa
+  instrumentação para bibliotecas e estruturas de trabalho comuns
+- Componentes de instrumentação automática que geram dados de telemetria sem
+  exigir alterações no código
+- O [OpenTelemetry Collector](/docs/collector), é um intermediário (_proxy_) que
+  recebe, processa e exporta dados de telemetria
+- Várias outras ferramentas, como o
+  [OpenTelemetry Operador para Kubernetes](/docs/kubernetes/operator/),
+  [OpenTelemetry Helm Charts](/docs/kubernetes/helm/), e
+  [comunidade ativa para FaaS](/docs/faas/)
+
+O OpenTelemetry é utilizado por uma grande variedade de
+[bibliotecas, serviços e aplicativos](/ecosystem/integrations/) que o integram
+para fornecer um padrão de observabilidade.
+
+OpenTelemetry é suportado por inúmeros [fornecedores](/ecosystem/vendors/),
+muitos dos quais oferecem suporte comercial para OpenTelemetry e contribuem
+diretamente para o projeto.
+
+## Extensibilidade
+
+O OpenTelemetry é projetado para ser extensível. Alguns exemplos de como ele
+pode ser estendido incluem:
+
+- Adicionar um receptor ao OpenTelemetry Collector para suportar dados de
+  telemetria de uma fonte personalizada
+- Carregar bibliotecas de instrumentação personalizadas em um SDK
+- Criar uma [distribuição](/docs/concepts/distributions/) de um SDK ou do
+  Collector adaptada a um caso de uso específico
+- Criar um novo exportador para um backend personalizado que ainda não suporta o
+  protocolo OpenTelemetry (OTLP)
+- Criar um propagador personalizado para um formato de propagação de contexto
+  não padrão
+
+Embora a maioria dos usuários possa não precisar estender o OpenTelemetry, o
+projeto é projetado para tornar isso possível em quase todos os níveis.
+
+## História
+
+O OpenTelemetry é um projeto
+[Cloud Native Computing Foundation (CNCF)](https://www.cncf.io) que é resultado
+da fusão entre dois projetos anteriores, [OpenTracing](https://opentracing.io)
+and [OpenCensus](https://opencensus.io).
+
+Ambos os projetos foram criados para resolver o mesmo problema: a falta de um
+padrão de como instrumentar o código e enviar dados de telemetria para um
+backend de Observabilidade. Como nenhum dos projetos conseguiu resolver o
```

Comment:
```suggestion
backend de Observabilidade. Como nenhum dos projetos conseguiu totalmente resolver o
```

We missed this important word.

Reply:
To me, it is not sound good: `conseguiu totalmente resolver`, maybe: `Como nenhum dos projetos conseguiu resolver o problema por completo.`

Reply:
Perfeito!